releasenotes/notes/rabi-and-qv-bugfix-34636baee6651af1.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Rabi experiment amplitudes are casted to `float` type for serialization purposes.
+  - |
+    QV experiment probabilities are casted to `list` type for serialization purposes.
```

Comment:
I would be more explicit about what this is actually fixing from the user's perspective. Something like, "Fixed a serialization bug where Rabi experiments did not run successfully through `qiskit-ibm-provider` when custom amplitudes were provided in a numpy array" and "Fixed a bug where QV experiments did not run successfully through `qiskit-ibm-provider` when Qiskit Terra was used to calculate ideal probabilities instead of Aer."

Reply:
I changed it to the following:
```
fixes:
  - |
    Resolved a serialization issue that affected Rabi experiments when running it through 
    `qiskit-ibm-provider` using custom amplitudes provided as a numpy array.
  - |
    Resolved an issue that caused QV experiments to fail when executed via `qiskit-ibm-provider` using
    Qiskit Terra for calculating ideal probabilities, instead of Aer.
```
Do you think this is OK?

Reply:
Looks good, thanks.